internal/server/data/data.go

```diff
@@ -21,6 +21,19 @@ import (
 )
 
 func NewDB(connection gorm.Dialector) (*gorm.DB, error) {
+	db, err := newRawDB(connection)
```

Comment:
We could make the name imply that it runs migration (`InitializeDB` ?), but maybe a godoc comment would be enough?

```suggestion
// NewDB creates a new database connection and runs any required database migrations
// before returning the connection.
func NewDB(connection gorm.Dialector) (*gorm.DB, error) {
	db, err := newRawDB(connection)
```

Reply:
yeah, since that includes automigrate and the memory db is usually empty, it's pretty useless to return a non-migrated db.. except in the case where you're trying to test the migrations! :D